AwesomeProject/screens/Menu/Menu.screen.js

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import {createNativeStackNavigator} from '@react-navigation/native-stack';
+import {NavigationContainer} from '@react-navigation/native';
+import {Button, View, ScrollView} from 'react-native';
+import styles from '../Menu/Menu.styles';
+import AboutMe from '../AboutMe/AboutMe.screen';
+
+export default Screens = () => (
+  <NavigationContainer>
+    <Stack.Navigator initialRouteName="Screens">
+      <Stack.Screen name="Menu" component={MenuScreen} />
+      <Stack.Screen name="Alexey Kundro" component={DetailsScreen} />
```

Comment:
if you want to pass you name to the title use options property for the screen. Find more information about it inside react-navigation api for StackNavigator